jsonnet/telemeter/prometheus/kubernetes.libsonnet

```diff
@@ -41,6 +41,7 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
       serviceAccount.new('prometheus-' + $._config.prometheus.name) +
       serviceAccount.mixin.metadata.withNamespace($._config.namespace) +
       serviceAccount.mixin.metadata.withAnnotations({
+        'serviceaccounts.openshift.io/oauth-redirectreference.observatorium-thanos-querier':'{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"observatorium-thanos-querier"}}',
```

Comment:
Can we add a TODO here to eliminate this once we have a real clusterrole?

Reply:
done! :+1: